helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -957,18 +957,27 @@ private RollingUpdateOp rollingUpdateAwaitRunning(final ZooKeeperClient client,
 
   private boolean isRolloutTimedOut(final ZooKeeperClient client,
                                     final DeploymentGroup deploymentGroup) {
+    final String groupName = deploymentGroup.getName();
+    final long groupTimeoutSetting = deploymentGroup.getRolloutOptions().getTimeout();
```

Comment:
I guess if NPE is here, we'll see it in logs?

Reply:
yes. this unboxing already existed, but as far as I can tell this value should never actually be null (I think it would have been preferable if the internal representation could be some type without nullable fields).